features/my_google.feature

```diff
@@ -10,17 +10,27 @@ Feature: As a web automation tester in training I want to automate basic searchi
     #A Then step tests the results of the action
     Then I see the Google Search Results Page
     #An And step mimics the purpose of the step above it, so this would function as another Then step
-    And the Google Search Results Page has Kittens
-
+    And the <page_name> Page has Kittens
     #You can perform any number of additional When-Then tests within a given scenario
-#    When I click the first result on the Google Search Results page
-#    Then I am redirected to the corresponding website from the search result
+    When I click the first result on the Google Search Results page
+    Then I am redirected to the corresponding website from the search result
+
+    Examples:
+    | page_name   |
+    | search_results  |
 
   #A new scenario for a News search, note that we're re-using a couple of earlier steps
   #Cucumber steps are defined once but can be reused all over
-  Scenario: Basic search, then switch to News
+  Scenario Outline: Basic search, then switch to News
     Given I am on the Google Home Page
     And I search for Kittens
     When I click the News Tab on the Google Search Results Page
     Then I see the Google News Results Page
-    And the Google News Results Page has Kittens
\ No newline at end of file
+    And the <page_name> Page has Kittens
+    When I click the first result on the Google News Results page
+    Then I am redirected to the corresponding website from the news results
+
+    Examples:
```

Comment:
As above, this could just be a regular scenario since your step def is going to capture page names anyways, and we really don't want to iterate this test twice.

```ruby
Scenario: Basic search, then switch to News
  Given I am on the Google Home Page
  And I search for Kittens
  When I click the News Tab on the Google Search Results Page
  Then I see the Google News Results Page
  And the News Results Page has Kittens
  ...
```